descriptors/ModuleDescriptor-template.json

```diff
@@ -8,7 +8,7 @@
     },
     {
       "id": "users",
-      "version": "15.2"
+      "version": "15.3"
```

Comment:
The latest release is still at 15.2. Please revert this:https://github.com/folio-org/mod-users/blob/v17.2.2/descriptors/ModuleDescriptor-template.json#L7